definitions/features/foreman_service.rb

```diff
@@ -0,0 +1,16 @@
+module ForemanMaintain
+  module Features
+    class ForemanService < ForemanMaintain::Feature
+      metadata do
+        label :puma
```

Comment:
I think you renamed everything but left this `puma`?

Reply:
My bad. Fixed!

Reply:
no worries, thank you for the quick fix